dockerfile

```
@@ -1,25 +1,31 @@
+
+# Build the Go app with CGO enabled for SQLite support
+RUN CGO_ENABLED=1 GOOS=linux go build -a -ldflags '-linkmode external -extldflags "-static"' -o main ./cmd/api
     
     # ---------- Final Stage ----------
    # Start a new stage from scratch
 FROM alpine:latest
 
-# Install necessary packages
-RUN apk --no-cache add ca-certificates
+# Install necessary packages including SQLite
+RUN apk --no-cache add ca-certificates sqlite
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)